default.nix

```diff
@@ -64,6 +64,14 @@ stdenv.mkDerivation {
          cmdliner
          angstrom
          batteries
+         ppxlib
+         ppx_import
+         ppx_sexp_conv
+         ppx_yojson_conv
+         ppx_deriving
+         sel
+         lsp
+         sexplib
```

Comment:
Is this change really meant to be included in this PR?

Reply:
@eponier ?

Reply:
No, it was a temp hack. Removed.